gourmet/plugins/import_export/website_import_plugins/foodnetwork_plugin.py

```diff
@@ -6,42 +6,49 @@ class FoodNetworkPlugin(PluginPlugin):
     target_pluggable = 'webimport_plugin'
 
     def test_url(self, url, data):
-        if 'foodnetwork.com' in url:
+        if 'foodnetwork.co.uk' in url:
             return 5
         return 0
```

Comment:
I know, you didn't introduce them, but is there a reason for the strange return code and would it be better to introduce an enum for this?

Reply:
I have no clue about it. There seem to be 3 different values: `0` (no match), `5` (match) and `4` (some unknown return code used by `cooksillustrated_plugin`). I will add an own file with a corresponding enumeration class.